client/utils.js

```diff
@@ -7,3 +7,8 @@ export function pathToRegExp( path ) {
 	}
 	return new RegExp( '^' + path + '(/.*)?$' );
 }
+
+export function isFullSiteEditingTheme( theme ) {
+	const features = theme?.taxonomies?.theme_feature;
+	return features && features.some( ( feature ) => feature.slug === 'block-templates' );
+}
```

Comment:
This file, am I right? It's older than me!

I kinda feel like this is not the most appropriate place to add a utility function — and I'm sort of backed by the fact that nobody touched it in years! 😄 

What about the `client/lib` folder?
It's full of single-function folders, but I would be ok if we created a `full-site-editing` folder that could maybe contain several FSE-related utilities in the future.

Alternatively, and arguably even better, it would be cool to have this function closer to where it's used.
I presume that you moved it here because it's used by two different routes (`themes` and `theme`), but it happens quite often for a route to import stuff from another.
For example, I can see lots of components from `themes` used in `theme`.
What about moving it into a new `client/my-sites/themes/is-full-site-editing-theme.js` file?

Reply:
> I presume that you moved it here because it's used by two different routes (themes and theme)

Yep that's exactly why.

> What about moving it into a new client/my-sites/themes/is-full-site-editing-theme.js file?

> it happens quite often for a route to import stuff from another.

Happy to do this. I find it to be more intuitive to have shared functions live in the closest possible _ancestor_ in the directory structure, but I recognize that it's an opinion, and I'm totally fine following existing conventions. I'll make the changes.

Reply:
Yeah I know what you mean.
I'm rusty with Calypso conventions at this point, so I'm kinda guessing that, given there are no shared utilities in the ancestor (`client/my-sites`), and given the function seems a bit too specific to be included in the main utilities folder (`client/lib`),  it _should_ be fine to just put it into one between `themes` and `theme` and call it a day.

But most importantly: I'd avoid `client/utils.js`. 😄 
I don't have any historical knowledge about it, but I'd argue that such a generic file, if left unchecked, could have easily spun out of control, getting chock full of random one-off functions.

Reply:
Ah and I just realized, like you pointed out, that `client/my-sites` would've been the best place to put the shared function given my preferences 😅

Either way, I'll go ahead and move things to a new `client/my-sites/themes/is-full-site-editing-theme.js`

Reply:
Done in 8807b10